### PR TITLE
Get key functionality

### PR DESCRIPTION
**Pull request by ArtemAnoshin**

### CODE REVIEW

File: lib/Cleantalk/ApbctWP/State.php [hunk 1]
```diff
@@ -430,7 +431,8 @@ protected function init()
         // Standalone or main site
         $this->api_key        = $this->settings['apikey'];
         $this->dashboard_link = 'https://cleantalk.org/my/' . ($this->user_token ? '?user_token=' . $this->user_token : '');
-        $this->notice_show    = $this->data['notice_trial'] || $this->data['notice_renew'] || $this->data['notice_incompatibility'] || $this->isHaveErrors();
+        $this->notice_show    = $this->data['notice_trial'] || $this->data['notice_renew'] || $this->isHaveErrors();
+        $this->account_email  = $this->data['account_email'];
```
Comment (Glomberg) [inline lib/Cleantalk/ApbctWP/State.php:435]: This property ($apbct->account_email) is not used in the code. So this assignment is unnecessary.

File: inc/cleantalk-settings.php [hunk 2]
```diff
@@ -2324,6 +2336,123 @@ function apbct_settings__get_key_auto($direct_call = false)
     }
 }
 
+function apbct_settings__update_account_email()
+{
+    global $apbct;
+    
+    $account_email = isset($_POST['accountEmail']) ? $_POST['accountEmail'] : false;
```
Comment (safronik) [inline inc/cleantalk-settings.php:2336]: Using Post::get( 'accountEmail' ) will be better

File: lib/Cleantalk/ApbctWP/State.php [hunk 3]
```diff
@@ -430,7 +430,7 @@ protected function init()
         // Standalone or main site
         $this->api_key        = $this->settings['apikey'];
         $this->dashboard_link = 'https://cleantalk.org/my/' . ($this->user_token ? '?user_token=' . $this->user_token : '');
-        $this->notice_show    = $this->data['notice_trial'] || $this->data['notice_renew'] || $this->isHaveErrors();
+        $this->notice_show    = $this->data['notice_trial'] || $this->data['notice_renew'] || $this->data['notice_incompatibility'] || $this->isHaveErrors();
```
Comment (safronik) [inline lib/Cleantalk/ApbctWP/State.php]: Why this is in this branch? o_O

File: lib/Cleantalk/Common/Compatibility.php [hunk 4]
```diff
@@ -87,4 +94,9 @@ public function w3tcCheckLateInitCallback()
 
         return false;
     }
+
+    function wpRocketCallback()
```
Comment (safronik) [inline lib/Cleantalk/Common/Compatibility.php]: Why this is in this branch? o_O

**Review by safronik — COMMENTED** [review]
Approved!

Remarks:
1. Few patches in one branch.
2. We should use Post::get( 'param' ) instead of direct call to $_POST.

**Pull request merged**